src/main/java/org/gridsuite/network/map/NetworkMapService.java

```diff
@@ -382,10 +389,13 @@ private static LoadMapData toMapData(Load load) {
     private static ShuntCompensatorMapData toMapData(ShuntCompensator shuntCompensator) {
         Terminal terminal = shuntCompensator.getTerminal();
         ShuntCompensatorMapData.ShuntCompensatorMapDataBuilder builder = ShuntCompensatorMapData.builder()
-            .name(shuntCompensator.getNameOrId())
-            .id(shuntCompensator.getId())
-            .terminalConnected(terminal.isConnected())
-            .voltageLevelId(terminal.getVoltageLevel().getId());
+                .name(shuntCompensator.getNameOrId())
+                .id(shuntCompensator.getId())
+                .maximumSectionCount(shuntCompensator.getMaximumSectionCount())
+                .bPerSection(shuntCompensator.getModel(ShuntCompensatorLinearModel.class).getBPerSection())
```

Comment:
Pb if the model is not a 'linear' model, but is a 'non linear' model ...

Reply:
fixed